### ISSUE TITLE
Issues/points discovered in formatting

### ISSUE THREAD

**Issue by xixiaofinland** [aheber/tree-sitter-sfapex#34]
A general thread to put all other items in here.

**Comment by xixiaofinland** [comment]
it seems `soql_query` and `sosl_query` are unnecessary. The direct child has exactly the same content.

```
  {
    "type": "soql_query",
    "named": true,
    "fields": {},
    "children": {
      "multiple": false,
      "required": true,
      "types": [
        {
          "type": "soql_query_body",
          "named": true
        }
      ]
    }
  },
```

**Comment by aheber** [comment]
You're right on the query nodes being redundant. The question is what to do about it. Dropping them would be a breaking change and I don't know everything that could be impacted.

In the end, probably better to do it now than later, I'll mull it over before I commit but please send your thoughts on breaking changes like this.

(the unfortunate fact is that most changes in a tree-sitter project are breaking, this is just more breaking than I'd normally accept straight away)
